package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "slimerjs-core",
+  "version": "1.0.0",
```

Comment:
Which version should we use? Do you want to track against the latest version of Firefox that this package works with?